spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/LoadBalancerRetryPolicy.java

```diff
@@ -32,27 +32,71 @@
 	 * @param context the context for the retry operation
 	 * @return true to retry on the same service instance
 	 */
+	@Deprecated
```

Comment:
Please add JavaDoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added